simtools/io_handler.py

```diff
@@ -276,6 +294,21 @@ def getTestModelFile(fileName):
     return directory.joinpath(fileName)
 
 
+def getTestDerivedDirectory():
```

Comment:
Is there a motivation why you hear do the path mkdir in this function, while in getDerivedOutputDirectory, `_getOutputDirectory(filesLocation, label, "derived")` is called?

Reply:
I copied the way you wrote the other `getTest*Directory()` functions. I think it is because it is because it makes the tests simpler since we do not need to call `cfg.get("outputLocation")`. However, I must admit this is a question I should have asked you when you added those functions.